### PR TITLE
Update readme, laravel section

### LINKED ISSUE #169: Add note pointing to new laravel-broadcaster
- Replace old laravel description pointing to new laravel-broadcaster.

### PR DESCRIPTION
**Pull request by sacOO7**
- Updated laravel section with `laravel-broadcaster`
- Updated release process
- Fixes #169

### CODE REVIEW

**Review by owenpearson — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -135,9 +135,12 @@ $statsPage->next(); // retrieves the next page => \Ably\Models\PaginatedResult
 $client->time(); // in milliseconds => 1430313364993
 ```
 
-## Laravel
+## Laravel realtime broadcasting is now officially supported!
+
+If you're using Laravel and want to support **realtime broadcasting and events**, you may want to check out [laravel-broadcaster](https://packagist.org/packages/ably/laravel-broadcaster/).
+
+>If you want **ably-php** as a rest dependency across service providers, check [ably-php-laravel](https://packagist.org/packages/ably/ably-php-laravel). **ably-php-laravel** is a simple wrapper over **ably-php** with laravel-specific classes. This has limited usecases and **laravel-brodcaster** is recommended over **ably-php-laravel** for most-cases.
```
[the remark this answers is not in the document]
Comment (sacOO7) [reply README.md]: updated!

File: README.md [hunk 2]
```diff
@@ -135,9 +135,12 @@ $statsPage->next(); // retrieves the next page => \Ably\Models\PaginatedResult
 $client->time(); // in milliseconds => 1430313364993
 ```
 
-## Laravel
+## Laravel realtime broadcasting is now officially supported!
```
[the remark this answers is not in the document]
Comment (sacOO7) [reply README.md]: updated!

File: README.md [hunk 3]
```diff
@@ -135,9 +135,12 @@ $statsPage->next(); // retrieves the next page => \Ably\Models\PaginatedResult
 $client->time(); // in milliseconds => 1430313364993
 ```
 
-## Laravel
+## Laravel realtime broadcasting
+
+If you're using Laravel and want to support **realtime broadcasting and events**, you may want to check out [laravel-broadcaster](https://packagist.org/packages/ably/laravel-broadcaster/).
+
+> If you want **ably-php** as a rest dependency across service providers, check [ably-php-laravel](https://packagist.org/packages/ably/ably-php-laravel). **ably-php-laravel** is a simple wrapper over **ably-php** with laravel-specific classes. This has limited use-cases and **laravel-brodcaster** is recommended over **ably-php-laravel** for most use-cases.
```
Comment (qsdstefan) [inline README.md:142]: brodcaster -> broadcaster

**Review by qsdstefan — CHANGES_REQUESTED** [review]
Please fix the typo and I'll happily approve

File: README.md (hunk 3 above)
Comment (sacOO7) [reply README.md]: updated!

**Review by owenpearson — APPROVED** [review]
LGTM

**Review by qsdstefan — APPROVED** [review]
LGTM!

**Review by AmirQSD — APPROVED** [review]
LGTM

**Pull request merged**